### PR TITLE
Show Internal ID and Add Internal ID to CGH-Download

### PR DESCRIPTION
**Pull request by mikaell**
Ref: #1686

This PR adds a functionality or fixes a bug.

**How to test**:
1. how to test it, possibly with real cases/data

Install `feature-#1686_show_internal_name-200707`.
a. Go to Case page:
-Internal ID should show
b. Download CGH
-File name should be <case-id>.<internal-id>.vcf2cytosure.cgh

**Expected outcome**:
The functionality should be working

image

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
@mikaell let me know when this is ready for review so I can do it!

**Comment by mikaell** [comment]
> @mikaell let me know when this is ready for review so I can do it!

I think it's ready for review 😀

**Comment by northwestwitch** [comment]
Testing this on stage. It took me quite some time but I've found a case with the downloadable VCF2cytosure file there: https://scout-stage.scilifelab.se/cust000/643594-300M.

- The internal _id is displayed correctly on the case page. 🆗 
- The downloaded file contains **individual_name.pet_name.vcf2cytosure.cgh**

Now that I see it I think that users would be more happy if the downloaded file contains also the case display name, because that is the one they are used to see all the time.

I'll make a suggestion to the code so that the file will be named like this: **individual_name.case_display_name.pet_name.vcf2cytosure.cgh**

File: scout/server/blueprints/cases/views.py [hunk 1]
```diff
@@ -933,14 +933,11 @@ def vcf2cytosure(institute_id, case_name, individual_id):
...
-
+    institute_obj, case_obj = institute_and_case(store, institute_id, case_name)
     outdir = os.path.abspath(os.path.dirname(vcf2cytosure))
     filename = os.path.basename(vcf2cytosure)
-
-    LOG.debug("Attempt to deliver file {0} from dir {1}".format(filename, outdir))
-
-    attachment_filename = display_name + ".vcf2cytosure.cgh"
-
+    attachment_filename = display_name + "." + case_obj["_id"] + ".vcf2cytosure.cgh"
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/views.py:939]:
```suggestion
    attachment_filename = ".".join([ display_name, case_obj["display_name"], case_obj["_id"],  "vcf2cytosure.cgh" ])
```

**Pull request merged**